developer-docs-site/docs/guides/move-guides/move-scripts.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Move Scripts"
+slug: "move-scripts"
+---
+
+# Move Scripts
+
+You can use Move scripts to conduct arbitrary tasks on chain. 
+
+## Example use case
+
+The following example calls functions on the [aptos_coin.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/aptos_coin.move) module to confirm the balance of the destination account is less than `desired_balance`, and if so, tops it up to `desired_balance`.
+
+```move
+script {
+    use std::signer;
+    use aptos_framework::aptos_account;
+    use aptos_framework::aptos_coin;
+    use aptos_framework::coin;
+
+    fun main(src: &signer, dest: address, desired_balance: u64) {
+        let src_addr = signer::address_of(src);
+
+        let balance = coin::balance<aptos_coin::AptosCoin>(src_addr);
+        if (balance < desired_balance) {
+            aptos_account::transfer(src, dest, desired_balance - balance);
+        };
+
+        addr::my_module::do_nothing();
+    }
+}
+```
+
+This will end up in a distinct Move script file. Notice also how it calls a function in a Move module defined as:
+```
+module addr::my_module {
+    public entry fun do_nothing() { }
+}
+```
+
+That entry should go in a Move module file.
+
+Now you need to determine:
```

Comment:
Is it useful to list these questions here instead of directly below along with the answers?

Reply:
So for context, this is from a Stack Overflow Q&A. I like the Q&A format and honestly I think the partial rewrite into a guide / tutorial makes this weaker.

Reply:
Okay

Reply:
I'm open to changes, idk what's best now, the style of this doc has changed a lot now lol

Reply:
I moved the questions to be the intro of the Execution section (which was my gut response yet trying not to trample Daniel's fine work).